gallery/src/DarkModeModal/DarkModeModal.tsx

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useEffect, useState } from 'react';
+
+import {
+  NxButton,
+  NxModal,
+  NxFormGroup,
+  NxToggle,
+  NxFieldset,
+  NxRadio,
+  NxTooltip,
+  NxFontAwesomeIcon,
+  NxCode
+} from '@sonatype/react-shared-components';
+
+import { faInfoCircle } from '@fortawesome/free-solid-svg-icons';
+
+import { themingEnabled, themeOverride, setThemingEnabled, setThemeOverride } from '../themeClassManager';
+
+type Props = {
+  modalCloseHandler: () => void;
+};
+
+const DarkModeModal = (props:Props) => {
+  const { modalCloseHandler } = props;
+
+  const [themingEnabledState, setThemingEnabledState] = useState(themingEnabled),
```

Comment:
I wonder if we could use `useToggle` on the themingEnabledState?

Reply:
> I wonder if we could use useToggle on the themingEnabledState?

changed to useToggle here: [9e94c49](https://github.com/sonatype/sonatype-react-shared-components/pull/782/commits/9e94c49f66560c7ce74771f733b5496b3582a7db)